Cabal-syntax/src/Distribution/Compat/NonEmptySet.hs

```diff
@@ -138,6 +148,7 @@ member x (NES xs) = Set.member x xs
 -- Map
 -------------------------------------------------------------------------------
 
+{- FOURMOLU_DISABLE -}
```

Comment:
How do you decide what to disable? (Similar question to what @ffaf1 asked here https://github.com/haskell/cabal/pull/8950#discussion_r1197804606)

Reply:
When Fourmolu complains about CPP.